### DISCUSSION TITLE
Simple example?

### DISCUSSION THREAD

**Discussion by SailingNut** [codetheweb/tuyapi#548] [Q&A] [answered]
Hi all, I'm just in the planning phase of trying to write some code on a Raspberry PI to control my lights. I was wondering if anyone has a very simple example of code that they would be willing to share.

I'm not really sure how to integrate this API into an overall solution. (I'm not a web programmer, I do more backend stuff and standalone executables.)

Thanks in advance for any and all help!

**Comment by kueblc** [comment]
Did you try the example scripts in the README?

**Comment by SailingNut** [comment]
My problem is that I'm not sure what "extra" I need to combine with the scripts to get them to work. I know probably an HTML page to embed the JS into but any specifics I need to include in the HTML etc.

I know, a VERY basic question. Sorry!

**Comment by kueblc** [comment, marked as answer]
Ah I see the confusion. This is not a browser library, there is no HTML. This runs on `nodejs`, a console based JS engine.

**Comment by SailingNut** [comment]
Duh on my part!

Thanks tons for the push in the right direction!
